airgun/widgets.py

```diff
@@ -20,8 +20,8 @@
 from widgetastic_patternfly import FlashMessages
 from widgetastic_patternfly import Kebab
 from widgetastic_patternfly import VerticalNavigation
+from widgetastic_patternfly4.button import Button as PF4Button
```

Comment:
All of these dialog buttons should have static ouiaID's.

https://github.com/theforeman/foreman/commit/4bcd67ddb14abad4c9b44e213e19e1dfda7c8b7e

I don't think this is going the right direction.

Reply:
```<button aria-disabled="false" class="pf-c-button pf-m-danger" type="button" data-ouia-component-type="PF4/Button" data-ouia-safe="true" data-ouia-component-id="btn-modal-confirm">Confirm</button>```

The buttons on the PF4 confirmation dialog modal have data-ouia-component-id's set, you should be able to use them.  Their inclusion is probably what appears to have 'broken' the modal. 

![image](https://user-images.githubusercontent.com/21315076/179211458-596bc593-fa62-4f60-8a20-0494756e5d53.png)

Reply:
I pinged in the UX channel about the close button, its not as simple to set an ouiaID on because we'll have to define the entire header. There is an ouia component ID for the close button though.